packages/core/src/components/graphs/skeleton.ts

```diff
@@ -170,7 +170,7 @@ export class Skeleton extends Component {
 		const height = this.backdrop.attr("height");
 		const width = this.backdrop.attr("width");
 		const ticksNumber = Tools.getProperty(
-			this.model.getOptions(),
+			this.getOptions(),
```

Comment:
there's no `this.getOptions()` in the Skeleton class, is there? Why would we not get options from the model?

Reply:
its in the component class so ive updated components to use it.
in future if someone is making their own combo using constructors, they can add custom options to a skeleton graph that are applied on top of globals.